monitor_version_scripts/update_constants.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+Old_constants_FILE=$(cd .. && pwd)/images/constants.yml
```

Comment:
any reason we are using absolute paths here?

Reply:
No, it works with relative path as well

Reply:
Updated